src/script.js

```diff
@@ -372,6 +372,8 @@ module.exports = {
   fromASM: fromASM,
   toASM: toASM,
 
+  number: require('./script_number'),
```

Comment:
@jprichardson not going to release this publically till I get an ACK from you on this